src/game.cpp

```diff
@@ -365,6 +365,7 @@ void Game_ApplyControls(std::shared_ptr<Entity> ent)
     cam_angles[1] += 2.2f * engine_frame_time * look_logic[1];
     cam_angles[2] += 2.2f * engine_frame_time * look_logic[2];
 
+    // FIXME: Duplicate code - do we need cam control with no world??

```

Comment:
I believe it's a leftover from built-in model viewer, which was removed some time ago (because WadMerger is way easier to use).

I'm not sure it's not completely needed, as theoretically, user can launch engine without pre-loaded level file. However, he won't see anything except splash screen in this case.